lib/board.rb

```diff
@@ -1,7 +1,11 @@
 class Board
   attr_accessor :values
   def initialize
-    @values = Array.new(3) { |row| Array.new(3) { |col| row * 3 + col + 1 }}
+    @values = Array.new(3) { |row| Array.new(3) { |col| row * 3 + col + 1 } }
+  end
+
+  def update(move, sign)
+    @values[(move - 1) / 3.floor][(move - 1) % 3] = sign
```

Comment:
Perhaps it would be better to directly call this `values` rather than `@values` as it's already indicated as `attr_accessor`. In relation to this, is there another way to accomplish what this method does without making your values instance variable writeable (ergo being in `attr_accessor`)?

Reply:
Yes, I think it doesn't have to be accessible with attr_accessor at all.